pkg/model/combine.go

```diff
@@ -23,29 +20,48 @@ var ObjectCombiner = objectCombiner{}
 var _ common.ObjectCombiner = (*objectCombiner)(nil)
 
 // Combine implements tempodb/encoding/common.ObjectCombiner
-func (o objectCombiner) Combine(dataEncoding string, objs ...[]byte) ([]byte, bool) {
+func (o objectCombiner) Combine(dataEncoding string, objs ...[]byte) ([]byte, bool, error) {
 	if len(objs) <= 0 {
-		return nil, false
+		return nil, false, errors.New("no objects provided")
```

Comment:
Just thinking something about including the method or action we're erroring on.  Something like "unable to combine zero objects" or something.  Unless the caller has the context, in which case perhaps it doesn't matter.